src/components/mote/components/innkalling/Moteplanleggeren.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ReactElement } from "react";
+import { MoteDTO } from "../../../../data/mote/types/moteTypes";
+import { SeMoteStatus } from "./SeMoteStatus";
+import { MoteIkonBlaaImage } from "../../../../../img/ImageComponents";
+import { DialogmotePanel } from "../DialogmotePanel";
+import { tilDatoMedUkedagOgManedNavn } from "../../../../utils/datoUtils";
+import { useAktivtMoteplanleggerMote } from "../../../../data/mote/moter_hooks";
+import { useFnrParam } from "../../../../hooks/useFnrParam";
+import { Knapp } from "nav-frontend-knapper";
+import { FlexRow } from "../../../Layout";
+import { useHistory } from "react-router";
+
+const texts = {
+  nyttMote: "Nytt dialogmøte",
+  dialogMote: "Dialogmøte",
+  moteforesporselSendt: "Møteforespørsel sendt",
+  bekreftetMote: "Bekreftet møte",
+  seMotestatus: "Se møtestatus",
+  planleggNyttMote: "Planlegg nytt dialogmøte",
+  ingenMoterPlanlagt: "Ingen møter planlagt",
+};
+
+const resolveUndertittelForMoteStatus = (mote: MoteDTO) => {
```

Comment:
Denne dupliserer vel tilsvarende funksjon i InnkallingDialogmotePanel. Bør vi refaktorere det ut i util e.l. eller er tanken at dette forsvinner når vi går i prod med ferdig løsning?

Samme gjelder det som returneres fra komponenten her (dupliserer return fra InnkallingDialogmotePanel). Kunne man endret på hvordan det sjekkes for erLokalEllerPreprod så man slipper dupliseringen?

Reply:
Tanken var å kunne jobbe "uhindret" med dm2, og når vi går live så sletter vi denne komponenten. Kan selvsagt  legge til gjenbruk, men tenkte det var greit å ha dm2 som en helt egen greie, da det blir litt veksling mellom nytt og gammelt der osv